catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -57,8 +60,6 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ContentDirectoryMonitor.class);
 
-  private static final Logger INGEST_LOGGER = LoggerFactory.getLogger(Constants.INGEST_LOGGER_NAME);
```

Comment:
❓ Why replace the ingest logger?

Reply:
I though it was unnecessary. We only logged when the route wasn't started, but it was already logged in the main logs too.